src/phases/execution/ClusteredCollection.yml

```diff
@@ -73,25 +75,24 @@ Actors:
             key: {_id: "hashed"}
             numInitialChunks: 12
 
-# Phase 2: insert. TODO Make multithreaded once TIG-2938 is resolved.
 - Name: Insert
   Type: Loader
-  Threads: 1
+  Threads: *LoadingThreads
   Phases:
     OnlyActiveInPhases:
       Active: [2]
       NopInPhasesUpTo: *MaxPhases
       PhaseConfig:
         Repeat: 1
         Database: *Database
-        Threads: 1
+        MultipleThreadsPerCollection: true
         CollectionCount: 1
         DocumentCount: *docCount
         BatchSize: 1000
         Document:
           # Start from {_id: "0000001"} in order to be able to set the ^RandomInt max bound to *docCount
           # in the LookupByClusterKey phase. This works around TIG-3759.
-          _id: {^Join: {array: [{ ^FormatString: {"format": "%07d", "withArgs": [{^Inc: {start: 1}}]}}, *PaddingGenerator]}}
+          _id: {^Join: {array: [{ ^FormatString: {"format": "%07d", "withArgs": [{^Inc: {start: -62499, multiplier: *DocsPerThread}}]}}, *PaddingGenerator]}}
```

Comment:
If I'm not mistaken since this is the 3rd phase, the multiplier will be multiplied by 3 right? So I think the start should account for that -187499? -187499 + 3 * 62500 = 1

Reply:
You are absolutely right, thanks! Just confirmed after changing:
```
MongoDB Enterprise > db.Collection0.find({}).sort({_id: 1}).limit(1)
{ "_id" : "0000001", "a" : "841163", "b" : "osPFriKJjijnLNDnsHAELMqjKgDLnJiHMjAMtErqKjCuBNrjILjFlqkEAvOtgpDMqODqpiMrAGttsABIhtCMDmLkjqrirhlmLuLoFNjmlFDtGnrEBLKIJnjPCMrAqCNvFmopgBIHGGrklhDpPrvEgsCOrEAviKqHLKECqGmGLAACDAviHsFPoLLJFGHLjmgngAMkKErsmNFuMGsGHkoCHgiOhGvmttiopgBEBFLLDCIluAjIOCGlhAtkgpjKHIBJmHBKnImrnvMkrGNtnGjgLrjDruPvlCtqBrFnllJPHjMiCsLgDnjHHvohAvkKnqrBiuLPgivPOsJHgjpgnujKtiFKNmtEKiMCPMBMgLnDMqiKMngDMOtuigKtMkoKrHsKPkDhJMEgotCAlnEmjEruDsuivpAtJLlMCgrrvohCNGjCIEruiuGimrJNoCumrgGCOiLnqiJhokmulIAvvmEGsBoGPpNmErspikAlHskrJLImuINmhFgIPOhIgkKvAkCiNBgANMglhHmpAKDgBtptGGEjLjMjBJnjniGpGhrAHDvOGshHKHNFNBtLmNgtOAIPmtiIGOqpnGuvtCuGmhumtJiIhluFJhtjEtGGLBusHrLpiAmFOJACsJDrKuiJIjBnHCJmkOqDJluvPANCjlEhKqNGlrsjJDvMpgDtBiJDlAvpAHvOuoECmNHnFGmBghnODuphqmsvmoMFrgLDinmlDBAGCIBFMvrLFvpmDhlEHjLiKKFnnpqDtJvEANhKutHppAqmjBIiNvOrnJPuPALPutJCPKCAKkMoCpEEDMjJPgMqjolvqsGFMAsBvmjMjJphoALOMIoCjGJosoLvItiLAKlnmFCIFDHJmiFNlgPqtgooOpoCmPpsgAskgkJitJqAnDtopLJsKIhMnvvOkgjpGIMGpAPmoqJvtBNvnGmECPtInspOKorkqphHDnoqKOIggulKuqFlrriMAiBIGiiMBFEsDvrCiqhoHNJuLNtIDAgLADDOjrAKlDtFvhMpKMHJ" }
MongoDB Enterprise > db.Collection0.find({}).sort({_id: -1}).limit(1)
{ "_id" : "1000000", "a" : "545650", "b" : "CngnMCqnPkIrItOHNLlIhjMHOrnrmMuqqMHpKGBClAPlrhpGIPJGuAICuqDMqBMFvFPGHjBEiLhNFGEuokKnCmLivojiOjupgpLDkNCgtMiupOpKlqoIqhtDGrBrtogGmuGsBpsKEhBhgKvtCIhNhvChNtJLGPIrABnPvNEgLomEBhtkMiKtAJHKLujniHnpMtgOpBMDAqhLMHEHAKGMpuoIqvjjqGljEKElGDgEHLAKIHrgogpFtoCvrHDiMiDMKjLIoPinBJGgPvhGPKljMKLlCKvMuttOtJElHEAEnLhnDOpoumhEtPhPmPJkmIjHtBKPJkLIgIKPlkjvsvAMAumgtOuCiqCiluDKqtFjMFrlFkmunPmKGhkGnlEovBvmvNOOmFLtnnIvPPmgnOgkEnFkKogKvgPuliBKCAJMDFEqLsPJmrvGhpPKFqHiogmtLhnBIprmEOCCuErrnkMjConGCvOFMMNLvLOAKtCrjLDIBGOCkjhlhnKtGEtqOGDgCDuHMONtFOKvJMqMBtthLmKpDPhErBClCIuMFIKGpCAPiBLDlEIrPHgAjuJuODmElnkmHNKrjIJpqJNLKHGrCPmrBJsLBBvnhmMFkmvLEHKBKrOjvuNmjHGvjiFlEiMPoGIFvONkOMLljPHIgNnHgGrgJmgnsnototGFrOmHpoBouPNhpritntqtjCtKulHCCCBjLDEvkrhDMqioDFGOvDMLGhngqLIOKoEhkDqkDIMEglEFAGkBoiImNAELIILjDukKkHuErAJqoINDkjEgLqGglGEgJmCjpKGqGFEorDDrLtphKjNELoLEgjMgGAtvlOpIhPBkLGEJJpFpLltEvijoIhDOnqFMpMIiqgluuKjooIIolpEDDOGEANmMtiIknoJlkAkhIvmDtshFDDGhMBEEEiGEtgLMluuEGHokCmAFAqhqsLBPmuiDHuoHOMNPqMqosikqsvoCEFOhvMmMpGrNNNnGHAArCOluutFjJqsqnDoJrpMCJOHDqtkhgoJn" }
```